docs: update AppKit terminology to Reown across documentation (#738)
reown-com/reown-docs: snippets/appkit/shared/theming.mdx

> The theme for the Reown integration in your dApp can be fully customized.

Reviewer: ```suggestion
You can customize the theme of your Reown AppKit integration fully.
```
